build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -97,10 +96,7 @@
         "tensorflow::external_mhlo_includes",
         "MhloPasses",
         "MhloShapeOpsToStandard",
-        "MhloToArithmeticConversion",
```

Comment:
Can you add a note that we deliberately don't include all the passes because there are a bunch we don't use?

Reply:
Good idea, thanks for the suggestion